shops/src/main/resources/plugin.yml

```diff
@@ -4,4 +4,4 @@ name: Shops
 load: POSTWORLD
 version: '1.0'
 depend: [Core]
-api-version: 1.20
\ No newline at end of file
+api-version: '1.20'
```

Comment:
Not sure this change is needed (see other plugin.yml in the repo)

Reply:
The other plugin.yml's use api-version 1.19. When Spigot loads this as a float number it will recognize it as a1.2 and not 1.20. 

This happens to anyone using 1.20 as number and not a string:
- https://www.spigotmc.org/threads/tutor-solving-unsupported-api-version-1-2-error-in-1-20-plugins.607378/
- https://stackoverflow.com/questions/76464042/error-loading-plugin-unsupported-api-version-1-2

Reply:
All good then